android/capacitor/src/main/java/com/getcapacitor/plugin/notification/LocalNotificationManager.java

```diff
@@ -166,6 +166,28 @@ private void buildNotification(NotificationManagerCompat notificationManager, Lo
 
     mBuilder.setSmallIcon(localNotification.getSmallIcon(context));
     createActionIntents(localNotification, mBuilder);
+    
+    String text = localNotification.getBody();
+
+    if (text != null) {
+      if (text.contains("\n")) {
+
+        NotificationCompat.InboxStyle style = new NotificationCompat.InboxStyle((mBuilder));
+
+        for (String line : text.split("\n")) {
+          style.addLine(line);
+        }
+
+        mBuilder.setStyle(style);
+
+
+      } else {
+
+        mBuilder.setStyle(new NotificationCompat.BigTextStyle().bigText(localNotification.getBigText()));
```

Comment:
CI is failing because of missing method localNotification.getBigText()
should it be the following?:
mBuilder.setStyle(new NotificationCompat.BigTextStyle().bigText(text));